### DISCUSSION TITLE
Shortcuts in folder-based routing by using path as folder-name

### DISCUSSION THREAD

**Discussion by Glitchy-Tozier** [sveltejs/kit#6526] [Ideas]
Is it possible in folder-based routing to take a shortcut by naming folders like this "`abcd/xyz` " instead of first creating the folder `abcd` and inside creating `xyz`?

Maybe there's some other solution I didn't think about, but with my structure, this seems like this would be a great solution for projects that have to structure their links in the following way:

```
[slug]/edit/[slug]
[slug]/update/[slug]
[slug]/links/[slug]
[slug]/vote-links/[slug]
[slug]/result/[slug]
```

**Comment by mskocik** [comment]
You cannot create folder with `/` character. So it's not possible.

**Comment by Glitchy-Tozier** [comment]
That makes a lot of sense, actually :see_no_evil:
